crates/cargo-wdk/src/providers/fs.rs

```diff
@@ -0,0 +1,75 @@
+use std::{
+    fs::{copy, create_dir, rename, File, OpenOptions},
+    io::{Read, Write},
+    path::{Path, PathBuf},
+};
+
+use mockall::automock;
+
+use super::error::FileError;
+
+/// Provides limited access to `std::fs` methods
+pub struct FS {}
+
+/// A Provider trait with methods for file system access
+#[automock]
+pub trait FSProvider {
```

Comment:
Rename to `FsProvider` to align with the standard pascal case casing in Rust for types

Reply:
This & all provider traits are removed now, refer https://github.com/microsoft/windows-drivers-rs/pull/306#discussion_r1988308885